tests/components/wallbox/test_select.py

```diff
@@ -33,14 +33,9 @@
 ]
 
 
-@pytest.mark.parametrize(("mode", "response"), TEST_OPTIONS)
-async def test_wallbox_select_solar_charging_class(
-    hass: HomeAssistant, entry: MockConfigEntry, mode, response
-) -> None:
-    """Test wallbox select class."""
-
-    await setup_integration_select(hass, entry, response)
-
+@pytest.fixture
+def mock_wallbox_patches():
```

Comment:
lets call this `mock_authenticate` or something, and move the EcoSmart ones out into the single test that needs them. that way we can also use this fixture in `test_wallbox_select_class_error`